pkgs/development/python-modules/ipfshttpclient/default.nix

```diff
@@ -80,6 +80,8 @@ buildPythonPackage rec {
     runHook postCheck
   '';
 
+  doCheck = false;
```

Comment:
Whats this?

Reply:
See: https://github.com/NixOS/nixpkgs/issues/185294 for more context